client-impl/simple-client/src/main/java/uk/gov/gchq/palisade/client/ConfiguredClientServices.java

```diff
@@ -96,26 +96,27 @@ public ConfigurationService getConfigService() {
     }
 
     protected CacheService createCacheService() {
-        return new Configurator(configService).createFromConfig(CacheService.class, config);
+        return Configurator.createFromConfig(CacheService.class, config);
     }
 
     protected ResourceService createResourceService() {
-        return new Configurator(configService).createFromConfig(ResourceService.class, config);
+        return Configurator.createFromConfig(ResourceService.class, config);
     }
 
     protected AuditService createAuditService() {
-        return new Configurator(configService).createFromConfig(AuditService.class, config);
+        return null;
+//        return Configurator.createFromConfig(AuditService.class, config);
```

Comment:
Why does this return null?

Reply:
I was part of my debugging stuff that I forgot to change back